vm-wasmi/src/vm.rs

```diff
@@ -174,6 +176,27 @@ where
         )
     }
 
+    #[cfg(feature = "cosmwasm_1_2")]
```

Comment:
Let's not introduce flags for that, we didn't release yet.

Reply:
okay sir, then I'm using it by deafult